### ISSUE TITLE
Publish on Nextcloud app store

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/logreader#27]
And adjust app store link in README

**Comment by icewind1991** [comment]
was waiting for @jancborchardt to stop nitpicking before tagging a new version :smile:

**Comment by jancborchardt** [comment]
:D @icewind1991 I only had 2 feedbacks ;D basically only one

**Comment by icewind1991** [comment]
Blocked by https://github.com/nextcloud/appstore/issues/390

**Comment by jancborchardt** [comment]
So how do we proceed on this regarding the linked issue? @MorrisJobke @LukasReschke – not use the tag, or not publish it?

Oh and yeah, the default_enable seems stupid as a tag any app could set, should indeed be a list in the server core, no?

**Comment by MorrisJobke** [comment]
> Oh and yeah, the default_enable seems stupid as a tag any app could set, should indeed be a list in the server core, no?

Yes, it should be a list in the server.

**Comment by LukasReschke** [comment]
Why should we push this to the app store? The app is already enabled shipped in the TAR and ZIP ball? We can think about this for 12 but IMO not for 11.

**Comment by daita** [comment]
@LukasReschke When cloning NC to build a dev/work env, it is quite confusing to have to look throw all the apps of the store (twive) only to remember that you have to download logreader manually. Which is ironic as logreader is mandatory on a dev env :]

**Comment by MorrisJobke** [comment]
> @LukasReschke When cloning NC to build a dev/work env, it is quite confusing to have to look throw all the apps of the store (twive) only to remember that you have to download logreader manually. Which is ironic as logreader is mandatory on a dev env :]

We usually ship this in our tar balls. And if you need any other non-core app, then you also need to clone this individually. Also on a dev environment you could do a tail -f data/nextcloud.log 😝

**Comment by daita** [comment]
The parsing on logreader is reall impressive. the app should have a live update feature :]

**Comment by daita** [comment]
@MorrisJobke welp, I just had 2 minutes of clear thinking : 

> tail -f data/nextcloud.log | jq .

**Comment by nickvergessen** [comment]
> the app should have a live update feature :]

Feel free to open a new issue, sounds useful.

I will close this one for now, since appstore + shipped does not work atm.
